src/subscription_manager/cache.py

```diff
@@ -111,11 +111,9 @@ def write_cache(self, debug=True):
             f.close()
             if debug:
                 log.debug("Wrote cache: %s" % self.CACHE_FILE)
-        except IOError, e:
-            if debug:
-                log.error("Unable to write cache: %s" %
-                        self.CACHE_FILE)
-                log.exception(e)
+        except IOError, err:
```

Comment:
Please use
```python
except IOError as err:
```
per http://python3porting.com/noconv.html#handling-exceptions

Don't worry about existing instances unless you are already modifying them, but for all that you changed in this PR, please fix.